Makefile

```
@@ -12,9 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: verify
+# If you update this file, please follow
+# https://suva.sh/posts/well-documented-makefiles
+
+.DEFAULT_GOAL:=help
+SHELL:=/usr/bin/env bash
+
+##@ Package
+
+.PHONY: build-debs build-rpms
+
+build-debs: ## Build debs
+	PACKAGE_TYPE="debs" ./build/package-in-docker.sh
+
+build-rpms: ## Build rpms
+	PACKAGE_TYPE="rpms" ./build/package-in-docker.sh
+
+##@ Verify
+
+.PHONY: verify verify-shellcheck
+
 verify: verify-shellcheck ## Runs verification scripts to ensure correct execution
 
-.PHONY: verify-shellcheck
-verify-shellcheck:
+verify-shellcheck: ## Runs shellcheck
 	./hack/verify-shellcheck.sh
+
+##@ Helpers
+
+.PHONY: help
+
+help:  ## Display this help
+	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

:+1:

```suggestion
	@awk ' \
		BEGIN { \
			FS = ":.*##" ; \
			col = "\033[36m" ; \
			nocol = "\033[0m" ; \
			printf "\nUsage:\n  make %s<target>%s\n", col, nocol \
		} \
		/^[a-zA-Z_-]+:.*?##/ { \
			printf "  %s%-15s%s %s\n", col, $$1, nocol, $$2 \
		} \
		/^##@/ { \
			printf "\n%s%s%s\n", col, substr($$0, 5), nocol \
		} \
	' $(MAKEFILE_LIST)
```

Maybe that's a bit more readable? Anyway, I like that!